src/include/expression_evaluator/lambda_evaluator.h

```diff
@@ -59,6 +74,27 @@ class ListLambdaEvaluator : public ExpressionEvaluator {
         return result;
     }
 
+    std::vector<common::idx_t> getIndexInVars() {
```

Comment:
Let's put the implementation into cpp file. Some of the include will then be move to cpp as well.

Also since we use the terminology `param` everywhere. Maybe rename this function to `getParamIndices?`